src/main/resources/patchConfig.xml

```diff
@@ -18,7 +18,7 @@
     </module>
 
     <module name="com.puppycrawl.tools.checkstyle.filters.SuppressionPatchFilter">
-        <property name="file" value="/Users/hgh/Documents/GitHub/patch-filters/DiffReport/patch.txt"/>
+        <property name="file" value="${checkstyle.pathfilter.patch}"/>
```

Comment:
Update to `${checkstyle.patchfilter.patch}` (`c` is missed in `pathfilter` word)

Reply:
done